common/rya.api/src/main/java/mvm/rya/api/persist/index/RyaSecondaryIndexer.java

```diff
@@ -42,4 +44,14 @@
 
     public void dropGraph(RyaURI... graphs);
 
+    /**
+     * @return the set of predicates indexed by the indexer.
+     */
+    public abstract Set<URI> getIndexablePredicates();
```

Comment:
is this something that is uniform across secondary indexers?  For example, is nothing returned from PCJs?